README.md

```diff
@@ -1,2 +1,65 @@
 # Social-Distance-Checker
 Open Projects 2021
+
+
+![socialdistancing_mask_detection](https://user-images.githubusercontent.com/88554453/128603773-dceafea5-7f4f-46a8-8303-98dc23235368.jpg)
+
+# Abstract 
+
+The social distance checker is an opencv project in which we monitor the distances between the people and if the distance comes out to be less than the threshold value (calibrated six feet in terms of pixels) then we indicate the violation by surrounding them with a red bounding box and with the help of AI/ML/pretrained files , if the person does not wear a mask then his face gets surrounded by a red bounding box.
+
+# Motivation
+
+The Outbreak of COVID-19 has affected almost everyone across the globe. During this pandemic, people are 
+advised not to be in close contact with others to reduce the spread of the disease, but due to the negligence 
+of many, we continue to face the repercussions of this pandemic
+As the name suggests, social distancing implies that people should physically distance themselves from 
+one another, reducing the spread of contagious diseases like COVID. So, in order to monitor if people are 
+maintaining social distance or not, we are working on the project named, Social distance checker. 
+
+# Application
+
+It is used to maintain social distancing and helps in preventing the spread of deadly communicable diseases.
+
+# Limitations
+
+1.) It fails to detect the persons when the video of one of person gets blocked by another person. 
+
+2.) It fails to detect the depth between the persons due to several reasons like several times we dont know the focal length of the camera , etc.. 
+
+3.) As in different videos , the position of the camera is different so we have to calibrate threshold distance each time.
+
+# Future improvements
+
+1.) We can do the project with other softwares
+in the future like yolo ,better neural network ,whole 
+project on tenserflow 
+
+2.) We can integerate it with cctv to use it practically in daily life and we can use it with bird eye view method to get better results.
+
+# Team members
+
+1.) Alok Kohli
+
+2.) Grandhi Rahul Gupta
+
+3.) Prem Chhangani
+
+4.) Pallapu Sree Prudhveenadh
+
+5.) Vansh Mehra
+
+# Mentors
+
+1.) Agrim Aggarwal
+
+2.) Anfal Ansari
+
+3.) Annushree
+
+# References 
+
+1.) https://youtu.be/UM9oDhhAg88
+
+2.) https://youtu.be/Ax6P93r32KU
```

Comment:
Hyperlink the references properly. Mention the title for each reference and just include the URL in the hyperlink. Refer this README for more info: https://raw.githubusercontent.com/marsiitr/Rube-Goldberg-Machine/master/README.md